common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -479,6 +484,8 @@ def depth_first(subtree):
                         log.exception(
                             f'Course import {dest_id}: failed to import module location {child.location}'
                         )
+                        if self.status:
+                            self.status.fail(_(f'Failed to import module at location {child.location}'))
```

Comment:
I don't think these messages can be helpful for the enduser? what do you think?

Reply:
What do you suggest? I think the message points out the location so it should be helpful

Reply:
Let's discuss these.